### DISCUSSION TITLE
Improve "backup success" notification spam

### DISCUSSION THREAD

**Discussion by pcrockett** [nextcloud/all-in-one#3485] [Ideas] [answered]
### Is your feature request related to a problem? Please describe. 

My Nextcloud administrator account gets spammed with "backup success" notifications every day that I don't need. Daily notifications about successful backups are not only unnecessary to me; they also contribute to notification / alert fatigue, making it more difficult to separate signal from noise in my daily life.

_To be clear, I absolutely DO want to be notified about failed or skipped backups._

### Describe the solution you'd like

For other systems I administer, when possible, I send "job success" notifications to services like [Cronitor](https://cronitor.io/), [Dead Man's Snitch](https://deadmanssnitch.com/), or [Healthchecks.io](https://healthchecks.io/) via a simple `curl` command. These services then only send me a notification when they notice that the successful jobs STOP happening.

It would be ideal if Nextcloud could have a similar behavior: Only notify when backups stop being successful.

### Describe alternatives you've considered

* A solution that would be good enough (for me, at least) would be simply to delete [this block of code](https://github.com/nextcloud/all-in-one/blob/c4190e4560d369a6637f99184494d8134524fa6d/php/src/Cron/BackupNotification.php#L23C1-L25C2).
* I do remember a day [when I was a young warthog](https://www.youtube.com/watch?v=EILeWQEDtMw), when I did actually check these kinds of daily notifications. So I understand leaving it in there. In that case, it would be nice if there was a checkbox that would allow us to opt out.
* Add a new feature that allows the administrator to configure a healthcheck URL that gets pinged after every successful backup, which would allow a third party service to monitor the health of backups.

### Additional context

Would also involve some updates to the content under "Daily backup and automatic updates."

**Comment by szaimen** [comment, marked as answer]
Will be possible with https://github.com/nextcloud/all-in-one/pull/3516

**Reply by szaimen** [reply]
This is now released with v7.5.0 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel
